packages/react-core/src/components/Progress/Progress.tsx

```diff
@@ -113,7 +113,10 @@ export class Progress extends React.Component<ProgressProps> {
       );
     }
 
-    const scaledValue = Math.min(100, Math.max(0, Math.floor(((value - min) / (max - min)) * 100)));
+    const scaledValue = Math.min(
+      100,
+      Math.max(0, Math.floor(((value - min) / ((max <= min ? 100 : max) - min)) * 100))
```

Comment:
I kind of feel like if we are in this situation, we should just return 0. It seems like it would only happen if there is some kind of error in whatever is using this component.. or perhaps if it's still initializing some of the values or something? At this point it might be best just to see no progress as opposed to potentially some value that doesn't make sense.